src/Abstractions/NexusMods.Abstractions.Jobs/JobResultType.cs

```diff
@@ -8,15 +8,15 @@ public enum JobResultType : byte
     /// <summary>
     /// The job finished successfully.
     /// </summary>
-    Completed = 0,
+    Completed = 1,
 
     /// <summary>
     /// The job was cancelled.
     /// </summary>
-    Cancelled = 1,
+    Cancelled = 2,
 
     /// <summary>
     /// The job failed.
     /// </summary>
-    Failed = 2,
+    Failed = 3,
```

Comment:
Did you run into an issue with `default(JobResultType)` or what prompted this change? `JobStatus` has a `None = 0` value that I added as the "default" value. It makes somewhat sense semantically for the status, but it'd be odd on the `JobResultType` considering that it's just an enum for a tagged union.

Reply:
I thought there was a default issue here but it ended up being something else, I'll revert the change